src/layout/Body.js

```diff
@@ -0,0 +1,105 @@
+import React, { Fragment, Component } from 'react';
+import {
+    Bullseye,
+    Card,
+    CardHeader,
+    CardBody,
+    CardFooter,
+    Split,
+    SplitItem,
+    Grid,
+    GridItem,
+    TextContent,
+    Text,
+    TextVariants
+} from '@patternfly/react-core';
+import { ArrowRightIcon, BinocularsIcon } from '@patternfly/react-icons';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+class Body extends Component {
+    onLaunchClick = (event, name, url, title) => {
+        event.preventDefault();
+        localStorage.setItem('cs-app', name);
+        localStorage.setItem('cs-app-title', title);
+        window.location.href = `${document.baseURI}platform/${url}`;
+    }
+
+    render() {
+        const { technologies } = this.props;
+        return (
+            <Fragment>
+                <div>
+                    <Bullseye>
+                        <TextContent>
+                            <Text component={ TextVariants.h1 }>
+                                Explore Cloud Technologies
+                            </Text>
+                        </TextContent>
+                    </Bullseye>
+                </div>
+                <Grid sm={ 12 } md={ 6 } lg={ 3 } gutter="md">
+                    { technologies.map(({ icon: Icon, iconProps, title, url, body, isPreview, name }, key) => (
+                        <GridItem key={ key }>
+                            <Card className="ins-c-application-info">
+                                <CardHeader><Icon size="xl" className="ins-c-icon__active" { ...iconProps } /></CardHeader>
```

Comment:
These Icons will be replaced by Red Hat branded icons instead of Font Awesome/Patternfly Icons

Reply:
I opened a PR with proposed openshift icon in PF icons https://github.com/patternfly/patternfly-react/pull/1508. Honestly I think that it would be awesome if all of our brand and other icons were part of that lib. But I am gonna use svg imports here for now.

Reply:
That's fine. Eventually, we'll probably have to pull in the RH icons from the PNT portal (where all of the icons in the mocks came from)